### PR TITLE
Adds floortile camouflage equipment as hidden cargo crate

### PR DESCRIPTION
**Pull request by bigfatbananacyclops**
## About The Pull Request

hello I noticed the game hasn't gotten a good pr in 5 years so I decided to make one.

this is a pr I couldnt get merged long ago but now is the time.

basically it adds a crate(available through hacked module) containing a floortile camouflaged suit, shoes, gloves and balaclava.
x4 of each.

## Why It's Good For The Game

lets people have some extra fun with the useful cheap variety of stuff through hacked modules. i'd personally like to see this used for ops.

## THIS IS SO BROKEN I DONT WANNA GET KILLED BY THAT 1 TIME A YEAR.

it's extremely easy to spot with ambient occ activated, if they're moving too and also their eyes are the only thing uncovered.

## THIS SHOULD BE A TRAITOR ITEM!!!

there is a chameleon kit so there's no point in adding it, and it really doesn't have any benefit unless you stand still, regardless of that you can still be seen through ambient occlusion. It's really useless and no antag would like to carry a valid suit with a spare suit to change it when you can just get chameleon.

## THIS DOESNT MAKE SENSE!! NOT MRP COMPLIANT HOW COULD A FLOORTILE CAMOUFLAGE EVEN EXIST??!!

you could say this is the failed product of a chameleon suit. its not really a game-breaking thing it's a hallway styled camouflage suit. relax.

## Changelog

:cl:
add: floortile camouflage suit, boots, gloves and crate containing the former items
/:cl:

[
ftc_product
](url)

### CODE REVIEW

**Comment by Admiral-Gigaton** [comment]
This is great, this is terrible, you should make more masterpieces like this, and you need to have your fingers snipped off so you can't make anything like this again

**Comment by Loafin34** [comment]
this is brilliant. if only it adapted to the floortile under you...

**Comment by MrMelbert** [comment]
I think we could generate the sprite via a filter so we don't need to update all the sprites if/when floor tiles resprited. Probably doesn't need to be done in this PR though

File: code/modules/cargo/packs/imports.dm [hunk 1]
```diff
@@ -319,3 +319,20 @@
...
 	crate_type = /obj/structure/closet/crate/cargo
+
+/datum/supply_pack/imports/floortilecamo
+	name = "Floor-tile Camouflage Uniform"
+	desc = "Thank you for shopping from Camo-J's, our uniquely designed \
+		floor-tile 'NT SCUM' styled camouflage fatigues is the ultimate \
+		espionage uniform used by the very best. Providing the best \
+		flexibility, with our latest Camo-tech threads. Perfect for \
+		risky espionage hallway operations. Enjoy our product!"
+	hidden = TRUE;
```
Comment (MrMelbert) [inline code/modules/cargo/packs/imports.dm:330]:
```suggestion
	hidden = TRUE
```

File: code/modules/cargo/packs/imports.dm [hunk 2]
```diff
@@ -319,3 +319,20 @@
...
+	cost = CARGO_CRATE_VALUE * 6
+	contains = list(/obj/item/clothing/under/syndicate/floortilecamo = 4,
+			/obj/item/clothing/mask/floortilebalaclava = 4,
+			/obj/item/clothing/gloves/combat/floortile = 4,
+			/obj/item/clothing/shoes/jackboots/floortile = 4
+			)
```
Comment (MrMelbert) [inline code/modules/cargo/packs/imports.dm:336]:
```suggestion
	contains = list(
		/obj/item/clothing/under/syndicate/floortilecamo = 4,
		/obj/item/clothing/mask/floortilebalaclava = 4,
		/obj/item/clothing/gloves/combat/floortile = 4,
		/obj/item/clothing/shoes/jackboots/floortile = 4
	)
```

**Review by ZephyrTFA — APPROVED** [review]

**Comment by CRITAWAKETS** [comment]
This should be made available to nukies through some means since they can't get chameleon.
I dunno, 2-4TC kit in the same section as the centcom suit?

**Comment by bigfatbananacyclops** [comment]
>  should be made available to nukies through some means since they can't get chameleon.
> I dunno, 2-4TC kit in the same section as the centcom suit

there really is no point to it because you can see it with ambient occlusion. Even if it was filtered with the floor

**Comment by MrMelbert** [comment]
I like this better in Cargo

**Comment by Ghommie** [comment]
> hello I noticed the game hasn't gotten a good pr in 5 years so I decided to make one.

An enemy PR appeared: https://github.com/tgstation/tgstation/pull/81245

Anyway, good stuff I guess. I don't think many people having ambient occlusion off, but perhaps they should have a faint outline filter, for the few who do.

**Comment by bigfatbananacyclops** [comment]
> > hello I noticed the game hasn't gotten a good pr in 5 years so I decided to make one.
> 
> 
> 
> An enemy PR appeared: https://github.com/tgstation/tgstation/pull/81245
> 
> 
> 
> Anyway, good stuff I guess. I don't think many people having ambient occlusion off, but perhaps they should have a faint outline filter, for the few who do.
> 
> 

haha... if..

**Review by Wallemations — APPROVED** [review]
audibly laughed when I saw the picture, 100% approval lmao
I second that using a filter might be a good idea but I'm not familiar with how filters work, my job is just to make sure the sprites are ok

**Comment by Wallemations** [comment]
I have an additional recommendation;
Add this to the black market uplink
That thing doesn't get enough love imo

**Comment by Admiral-Gigaton** [comment]
addams-family-wednesday-addams

**Comment by bigfatbananacyclops** [comment]
pending merge..

**Comment by Wallemations** [comment]
I gave the sprites the OK but I don't see any other approvals by code maints so I can't merge it myself

**Comment by Wallemations** [comment]
Wait I'm blind

**Pull request merged**